Projects/AccountSDKIOSWebExample/ExampleWeb/ContentView.swift

```diff
@@ -63,7 +63,11 @@ struct ContentView: View {
                 
                 NavigationLink("", destination: webView, isActive: $showAccountPages)
             }
-        }.sheet(isPresented: $isPresentedSimplifiedLogin, content: { SimplifiedLoginSwiftUIView() })
+        }
+        // Please use .formSheet modifier defined in SDK to show Simplified Login UI. It takes care of showing it in the right size
```

Comment:
Maybe the SDK should take more responsibility of how to present 🤔 
Perhaps asking for the uiwindow as a parameter could be something to think about further ahead. Wdyt?

Reply:
I did it this way cause now we are returning ViewController and asking developers to show it modal instead of pushing to navigation view. So it's a more helper method for devs using our SDK in case of SwiftUI. But yes, maybe we need to find the solution to take full control of showing Simplified Login (but I think this .formSheet will be useful in that case also)